packages/mantine/src/components/collection/Collection.tsx

```diff
@@ -99,15 +100,32 @@ interface CollectionProps<T> extends DefaultProps<Selectors<typeof useStyles>> {
      * @default false
      */
     required?: boolean;
+    /**
+     * Label associated with a collection (title)
+     */
+    label?: string;
+    /**
+     * Description for the collection (under the label)
+     */
+    description?: string;
+    /**
+     * Error message if input is invalid
+     *
+     * @default 'Invalid Input'
+     */
+    error?: string;
 }
 
 const defaultProps: Partial<CollectionProps<unknown>> = {
     draggable: false,
     addLabel: 'Add item',
     addDisabledTooltip: 'There is already an empty item',
     disabled: false,
-    spacing: 'xs',
+    spacing: 'xl',
     required: false,
+    error: 'Invalid Input',
```

Comment:
```suggestion
    error: 'Invalid input',
```

I'm pretty sure someone in the Doc team would point that out 😆

Reply:
ok will fix